Meta/Lagom/CMakeLists.txt

```diff
@@ -485,6 +485,12 @@ if (BUILD_LAGOM)
             target_link_libraries(disasm LibCore LibELF LibX86 LibMain)
         endif()
 
+        if (NOT EMSCRIPTEN)
+            # LibELF is part of LibC in SerenityOS builds, but not in Lagom.
```

Comment:
If anyone expects this to be true, maybe we should just add LibELF to the link libraries of the dummy LibC target. However, the general concept of linking LibC instead of LibELF sounds weird already.

Reply:
What is "this" in "this to be true" here? The expectation that lagom and serenity builds link the same libs?

Reply:
Probably? There apparently is some special case here, since there is a comment pointing out the difference between on-Serenity and Lagom.

Reply:
I assume this is not merge blocking, the comment itself seems pretty clear to me.